pkgs/development/python-modules/wxpython/4.2.nix

```diff
@@ -61,6 +60,13 @@ buildPythonPackage rec {
     })
   ];
 
+  # https://github.com/wxWidgets/Phoenix/issues/2575
+  postPatch = ''
+    ln -s ${lib.getExe buildPackages.waf} bin/waf
```

Comment:
This is the first time I've ever seen `buildPackages`. Can you teach me what it's for (or more importantly, why we're using it here)? Why is this preferable to adding `waf` to the top? Is this a trick to reach "above/past" the python package set into the top level package set?

Reply:
This ensures that `waf` is native to the build system (so that it can be run during the build phase), and not to the host system, in a cross-compilation context.

Reply:
(I'm trying to mark this discussion resolved, but I cannot find the button...)